mungegithub/README.md

```diff
@@ -0,0 +1,109 @@
+    
+* **Release-note** Release note enforcement is another feather we are seeking from Mungegithub. With template, when prs are create, people should add release note (can be "None") in the pr description. Depended on the release message left, Mungegithub will add "release-note", "release-note-none". If you leave it empty, "release-note-needed" will be added and is going to block merging.
+
+* **do-not-merge** Even with all required parts, you can always have more time by add "do-not-merge" label. 
+
+### OWNERS file
+OWNERS file is the way to organize code owners and write priviliage. There are two parts. Reviews are the people who are suggested to review the pr and approvers are the people who can actually say "/approve" to add the "approve" label. Each path can have its OWNERS file, and this file affects this directory and all subdirectories. More details can be found: [Reviewer and approver](https://github.com/kubernetes/test-infra/tree/master/mungegithub/mungers/approvers)
+
+## Auto-merge a PR with Mungegithub
+
+### Stage One
+
+#### 1. Required CI Required CI status must be green. Due to the lack of github api, Mungegithub gets required CI from configration.
+[labels.png]
+
+#### 2. Four kinds of Labels "cla: yes", "lgtm", "approve", "release-ntoe"/"release-note-none"
+![alt tag](https://github.com/istio/test-infra/blob/mungegithub_doc/mungegithub/images/ci-status.png)
+
+* **cla** 
+
+  - Google-bot will add cla label, if you get a "cla: no" label, follow the instruction offered by googlebot to signup cla. 
+
+![alt tag](https://github.com/istio/test-infra/blob/mungegithub_doc/mungegithub/images/cla-label.png)
+
+* **lgtm** 
+
+  - Anyone in the organization can say `/lgtm` to add "lgtm" label.  
+  - No one can say `/lgtm` to his/her own PR.  
+  - When a valid approver say `/lgtm`, it implies `/approve` as well.  
+  - You can cancel "lgtm" by commenting `/lgtm cancel`, if you `/lgtm` before.
+
+![alt tag](https://github.com/istio/test-infra/blob/mungegithub_doc/mungegithub/images/lgtm.png)
+
+* **approved**
+
+  - When a person's name is in all OWNERS file(s) which is/are able to cover all changed files, he/she is a valid approver.
+A valid approver is able to say `/approve` and mungegithub will add "approved" label.  
+  - We already got ride of `/approve no-issue`, every PR can be approved by just saying `/approve`  
+  - A valid approver can say `/approve` to his/her own PR  
+  - You can cancel "approved" by commenting "/approve cancel", if you `/approve` before.
+
+![alt tag](https://github.com/istio/test-infra/blob/mungegithub_doc/mungegithub/images/approve.png)
+
+* **release-note**
+  - Use [PR template](https://github.com/istio/istio/blob/master/.github/PULL_REQUEST_TEMPLATE.md) to add release-note, make sure add "none" if that's not necessary for this PR.
```

Comment:
... to add release-note. **Make** sure **to** add "none" if **notes** not necessary for this PR.